### PR TITLE
Bundle generation improvements

### PR DESCRIPTION
**Pull request by phantomjinx**
- Upgrades tool versions
- Provides `pre-bundle` as separate rule to aid bundle generation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -91,44 +97,100 @@ endif
...
+CHANNELS ?= $(DEFAULT_CHANNEL),latest
+PACKAGE := hawtio-operator
+MANIFESTS := bundle
+CSV_VERSION := $(VERSION)
+CSV_NAME := $(PACKAGE).v$(CSV_VERSION)
+CSV_DISPLAY_NAME := Hawtio Operator
+CSV_FILENAME := $(PACKAGE).clusterserviceversion.yaml
+CSV_PATH := $(MANIFESTS)/bases/$(CSV_FILENAME)
+CSV_REPLACES := $(LAST_RELEASED_IMAGE_NAME).v$(LAST_RELEASED_VERSION)
+IMAGE_NAME ?= docker.io/hawtio/operator
```
Comment (tadayosi) [inline Makefile:123]: `quay.io`?

File: bundle/bases/hawtio-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -6,9 +6,10 @@ metadata:
     categories: Integration & Delivery
     certified: "false"
     description: Hawtio eases the discovery and management of Java applications deployed on OpenShift.
-    containerImage: docker.io/hawtio/operator
+    containerImage: docker.io/hawtio/operator:1.0.0
```
Comment (tadayosi) [inline bundle/bases/hawtio-operator.clusterserviceversion.yaml:9]: `quay.io`?

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -34,18 +33,22 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Integration & Delivery
     certified: "false"
-    containerImage: docker.io/hawtio/operator:0.5.0
-    createdAt: "2022-04-08T00:00:00Z"
-    description: Hawtio eases the discovery and management of Java applications deployed on OpenShift.
-    operators.operatorframework.io/builder: operator-sdk-v1.11.0+git
+    containerImage: docker.io/hawtio/operator:1.0.0
```
Comment (tadayosi) [inline bundle/manifests/hawtio-operator.clusterserviceversion.yaml:36]: `quay.io`?

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -150,7 +153,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: hawtio-operator
-                image: docker.io/hawtio/operator:0.5.0
+                image: docker.io/hawtio/operator:1.0.0
```
Comment (tadayosi) [inline bundle/manifests/hawtio-operator.clusterserviceversion.yaml:156]: `quay.io`?

**Review by tadayosi — APPROVED** [review]
LGMT. Just a few minor comments.

File: Makefile (hunk 1 above)
Comment (phantomjinx) [reply Makefile:123]: Will do a single upgrade of all mentions of docker.io -> quay.io in a separate PR. As you have seen, the mentions of docker.io are numerious and be good to catch them all at the same time.

File: bundle/bases/hawtio-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (phantomjinx) [reply bundle/bases/hawtio-operator.clusterserviceversion.yaml:9]: see above

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml (hunk 4 above)
Comment (phantomjinx) [reply bundle/manifests/hawtio-operator.clusterserviceversion.yaml:156]: see above

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (phantomjinx) [reply bundle/manifests/hawtio-operator.clusterserviceversion.yaml:36]: see above.

**Pull request merged**